src/scss/components/_media_box.scss

```diff
@@ -43,6 +43,7 @@
 
   &__title {
     color: inherit;
+    flex: 1;
```

Comment:
@eduardojmatos This prop always plays tricks on me on safari and IE 10,11. Did you validate it?

Reply:
@iondrimba tested on Safari 10. Testing now on IE 10|11...

Reply:
@iondrimba our `build:dev:css` task was ignoring postcss because the order of CLI commands it was wrong...

Fixing in this PR now.

Reply:
Correcting: `build:prod:css`. Our `build:dev:css` do not contains `postcss` for now... Adding here.

Reply:
@eduardojmatos Thats great! :1st_place_medal: :1st_place_medal: :1st_place_medal: :1st_place_medal: :1st_place_medal: :1st_place_medal: :1st_place_medal: :1st_place_medal: :1st_place_medal: :1st_place_medal: :1st_place_medal: